docs/development/maintenance.md

```diff
@@ -13,1142 +15,1290 @@ modules of ILIAS.
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de. Only relevant if 
-    **the maintenance_model** is set to "Classic" **and** neither a first nor a second maintainers is set.
-* **"coordinator"**: An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-	Only relevant if **the maintenance_model** is set to "Coordinator".
-* **"[pr_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** : 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"[issue_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** :
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, designers, testers and other contributors, but we also have to
+guarantee the sustainability and the quality of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: the people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign off Code Changes**
+2. **Authority to Sign off Conceptual Changes**
+3. **Authority to Sign off Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign off Conceptual
```

Comment:
Finden wir für dieses Modell noch einen anderen Namen, der nicht die konkrete Komponente adressiert? Vielleicht Collaboration Model oder etwas ähnliches. Dieses Modell sollte ja grundsätzlich auch für andere Komponenten nutzbar sein - und dann ist es doof, wenn hier das T&A hervorgehoben wird.

Reply:
Ich glaube nicht, dass wir hier einen "Typen" daraus machen sollten. In meinen Augen, ist dieses Beispiel genau hier, um einen massgeschneiderten Fall zu präsentieren, an dem mensch sich orientieren kann, der aber eben genau keine Blaupause ist. Wir könnten auch das WebDAV anbringen.

Reply:
Okay. Wobei ich mich auch gerade frage, ob man das T&A nicht auf Coordinator umstellen könnte mit Denis als einzigem Coordinator aber mehreren Entwicklern mit der "Licence to merge". Dann hätten wir hier keinen Sonderfall. Just my 2 pence...

Reply:
Ist wahrscheinlich ein bisschen haarspalterisch oder diskussionswürdig, aber: Aus meiner Perspektive ist der größte Unterschied zwischen dem, was Denis macht und dem was Timon und ich als Koordinatoren machen im "Authority to Sign off Conceptual Changes". IMHO nimmt Denis sich da (aus gute Gründen!) mehr Authority als Timon und ich das beim UI-Framework machen.

Reply:
Wir könnten ihn ja mal fragen, ob er sich auch als Coordinator sehen würde. Ich überlege in der Tat, den Language Service auf Coordinator umzustellen. Und würde mir trotzdem die authority zu Eigen machen.

Reply:
Mit der Änderung hier hat dies in meinen Augen eigentlich gar keine Bedeutung mehr. "Maintainer-Modell" und "Koordinatoren-Modell" sind nur noch Beschreibung/Vorlagen für eine soziale Organisation. Das T&A hat das T&A-Modell, WebDAV hat das WebDAV-Modell und die DataCollection hat das DataCollection-Modell. Diese Modelle basieren darauf, dass die Kompetenzen auf eine gewisse Art verteilt sind und dass die Verantwortlichen Regeln definiert haben, wie sie miteinander umgehen wollen. Daher ergibt es in meinen Augen nur wenig Sinn das T&A auf irgend etwas umzustellen.
Im T&A sollten wir dann bald einmal unsere soziale Organisation gut und öffentlich dokumentieren, aber eigentlich ist da schon einiges da, nur halt nicht verschriftlicht oder nicht öffentlich.